core/src/main/java/org/apache/iceberg/rest/RESTUtil.java

```diff
@@ -36,6 +38,7 @@ public class RESTUtil {
   private static final Joiner NAMESPACE_ESCAPED_JOINER = Joiner.on(NAMESPACE_ESCAPED_SEPARATOR);
   private static final Splitter NAMESPACE_ESCAPED_SPLITTER =
       Splitter.on(NAMESPACE_ESCAPED_SEPARATOR);
+  private static final SecureRandom SECURE_RANDOM = new SecureRandom();
```

Comment:
new SecureRandom() is relatively expensive; one shared instance is thread‑safe and sufficient.